app/assets/javascripts/components/utils.js.coffee

```diff
@@ -132,3 +132,9 @@ class @Utils
       .fail (xhr, status, error) ->
         alert('Error: ' + error)
         callback()
+
+  @select2TagClickHandler: (e, elem) ->
+    if e.which == 1
```

Comment:
It imitates the click behavior on normal links, left click opens it in the current window and middle click opens a new tab/window.
The problem is `select2` catches all clicks on the tags and stops the propagation, jquery `click` events only work when they are registered after `select2` rendered the tags, since I did not want to reinitialize the `click` callback after every `select2` update I went with the `onClick` handler.